src/components/themed/OutlinedTextInput.tsx

```diff
@@ -332,47 +324,48 @@ export const OutlinedTextInput = React.forwardRef<OutlinedTextInputRef, Outlined
             </View>
           </TouchableWithoutFeedback>
         ) : null}
-        {numeric ? (
-          <NumericInput
-            ref={inputRef}
-            {...inputProps}
-            minDecimals={minDecimals}
-            maxDecimals={maxDecimals}
-            autoFocus={autoFocus}
-            multiline={multiline}
-            editable={!showSpinner}
-            selectionColor={hasError ? theme.dangerText : theme.outlineTextInputTextColor}
-            style={[textStyle, textInputStyle]}
-            textAlignVertical="top"
-            value={value}
-            secureTextEntry={hidePassword}
-            // Callbacks:
-            onBlur={handleBlur}
-            onChangeText={onChangeText}
-            onFocus={handleFocus}
-            maxLength={maxLength}
-            onSelectionChange={handleSelectionChange}
-          />
-        ) : (
-          <TextInput
-            ref={inputRef}
-            {...inputProps}
-            autoFocus={autoFocus}
-            multiline={multiline}
-            editable={!showSpinner}
-            selectionColor={hasError ? theme.dangerText : theme.outlineTextInputTextColor}
-            style={[textStyle, textInputStyle]}
-            textAlignVertical="top"
-            value={value}
-            secureTextEntry={hidePassword}
-            // Callbacks:
-            onBlur={handleBlur}
-            onChangeText={onChangeText}
-            onFocus={handleFocus}
-            maxLength={maxLength}
-            onSelectionChange={handleSelectionChange}
-          />
-        )}
+        <View style={[styles.suffixContainer, suffixPadding]}>
```

Comment:
This affects ALL use of OutlineTextInput and I can't confirm this doesn't cause a regression. I would reduce the regression surface by limiting to textinput, and conditionalize to when `suffix` is being used. Even if it means copy and pasting the entire <TextInput> block, I'd be ok with that and prefer it to affecting all uses.